open_academy/models/course.py

```diff
@@ -4,6 +4,7 @@
 class Course(models.Model):
     _name = 'course'
     _description = 'Course'
+    identifier = fields.One2many('session', 'course_id')
```

Comment:
Why are you naming this field as `identifier`?, Please change it to `session_ids`